### PR TITLE
Feature/Add select to alumni map

### PR DESCRIPTION
**Pull request by Pei-Nie**

### CODE REVIEW

File: pages/applicants/origin-country.tsx [hunk 1]
```diff
@@ -46,19 +53,68 @@ const Page: NextPage<Props> = ({
...
-              >
-                <option value={""}>All levels</option>
-                {levels.map((d) => (
-                  <option value={d.level ?? ""} key={d.level}>
-                    {d.level}
-                  </option>
-                ))}
-              </select>
+              <Select.Root onValueChange={(value) => setLevel(value)}>
+                <Select.Trigger className="text-violet11 hover:bg-mauve3 data-[placeholder]:text-violet9 ml-4 inline-flex h-[35px] items-center justify-center gap-[5px] rounded bg-white px-[15px] text-[13px] leading-none shadow-[0_2px_10px] shadow-black/10 outline-none focus:shadow-[0_0_0_2px] focus:shadow-black">
```
Comment (jakoblistabarth) [inline pages/applicants/origin-country.tsx]: Regarding the styling: we don't have the tailwind utility-classes `mauve` and `violet`, they would need to be added to `tailwind.config.js` (see https://www.radix-ui.com/primitives/docs/components/select). Instead we want to make use of colors (utility classes) we use in other parts of the website. These are mainly `itc-green` and `itc-blue`. Please adjust the color-related classnames accordingly.

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "@paralleldrive/cuid2": "^2.2.0",
     "@prisma/client": "^5.2.0",
     "@radix-ui/colors": "^3.0.0",
+    "@radix-ui/react-icons": "^1.3.0",
```
Comment (jakoblistabarth) [inline package.json]: We already have react-icons as dependency (see below) – so no need to install it.
```suggestion
```

File: pages/applicants/origin-country.tsx [hunk 3]
```diff
@@ -21,6 +21,13 @@ import { getFilledSeries } from "../../components/LinePath/LinePath.helpers";
 import Section from "../../components/Section";
 import Container from "../../components/Container";
 import Paragraph from "../../components/Paragraph";
+import React from "react";
```
Comment (jakoblistabarth) [inline pages/applicants/origin-country.tsx]: I think this import is not required.
```suggestion
```

File: pages/applicants/origin-country.tsx [hunk 4]
```diff
@@ -21,6 +21,13 @@ import { getFilledSeries } from "../../components/LinePath/LinePath.helpers";
...
+import * as Select from "@radix-ui/react-select";
+import {
+  CheckIcon,
+  ChevronDownIcon,
+  ChevronUpIcon,
+} from "@radix-ui/react-icons";
```
Comment (jakoblistabarth) [inline pages/applicants/origin-country.tsx]: Please import equivalent icons from [react-icons](https://react-icons.github.io/react-icons/).

**Review by jakoblistabarth — CHANGES_REQUESTED** [review]

File: pages/applicants/origin-country.tsx (hunk 4 above)
Comment (Pei-Nie) [reply pages/applicants/origin-country.tsx]: Ok

Comment (Pei-Nie) [reply pages/applicants/origin-country.tsx:29]: ok.

**Review by jakoblistabarth — APPROVED** [review]

**Pull request merged**